pkg/bgpv1/manager/reconciler/service_test.go

```diff
@@ -740,14 +740,15 @@ func TestServiceReconcilerWithClusterIP(t *testing.T) {
 	svc1NonClusterIP.Spec.ClusterIP = "None"
 	svc1NonClusterIP.Spec.ClusterIPs = append(svc1NonClusterIP.Spec.ClusterIPs, "None")
 	svc1ETPLocal := svc1.DeepCopy()
-	svc1ETPLocal.Spec.ExternalTrafficPolicy = slim_corev1.ServiceExternalTrafficPolicyLocal
+	internalTrafficPolicyLocal := slim_corev1.ServiceInternalTrafficPolicyLocal
+	svc1ETPLocal.Spec.InternalTrafficPolicy = &internalTrafficPolicyLocal
```

Comment:
nit: svc1ETPLocal -> svc1ITPLocal

Reply:
Same for other variables in the test case, I am assuming it means ITP(InternalTrafficPolicy)Local.

Reply:
Updated.